### ISSUE TITLE
Crash on startup: Virtualized Windows 10 Pro 

### ISSUE THREAD

**Issue by pa3k-lu** [onivim/oni2#1522]
OS: Windows 10 Pro x64, 1909 build 18363.752
Oni2: Onivim 2.0.3.0 (97eb4fea) downloaded from EA

Environment:  Windows 10 Pro running in Proxmox 6.1, the same symptoms are in Windows 10 Pro virtualized in bhyve in FreeBSD 12.1-stable
  - with Microsoft Basic Display Driver and Microsoft Remote Display Driver
  - connected to admin console through RDP

Oni2 running from command line briefly opens white onivim window that after while closes, command:
```oni2 -f --debug --trace --log-file oni2.log```

oni2.log contains only:
```--
Log started at 2020-03-29T19:14:32Z
--

[INFO]  Oni2_editor Starting Onivim 2.0.3.0 (97eb4fea)
[DEBUG] Oni2_editor Calling App.start
[DEBUG] Oni2_editor Init
[DEBUG] Revery.Core.Window Starting window creation...
[INFO]  Revery.Core.Window Creating window Oni2 width: 800 height: 600
[INFO]  Revery.Core.Window Window created successfully.
[DEBUG] Revery.Core.Window Window id: 1
[DEBUG] Revery.Core.Window Setting window context
[DEBUG] Revery.Core.Window GL setup. Checking GL version...
[DEBUG] Revery.Core.Window Checking GL vendor...
[DEBUG] Revery.Core.Window Checking GL shading language version...
```

Windows Event Log contains error entries for Onivim2 crash:

```Faulting application name: Oni2_editor.exe, version: 0.0.0.0, time stamp: 0x5e80191d
Faulting module name: Oni2_editor.exe, version: 0.0.0.0, time stamp: 0x5e80191d
Exception code: 0xc0000005
Fault offset: 0x00000000006d460d
Faulting process id: 0x25c
Faulting application start time: 0x01d605ff8b8fec2d
Faulting application path: C:\Users\pa3k\Desktop\win32\Oni2_editor.exe
Faulting module path: C:\Users\pa3k\Desktop\win32\Oni2_editor.exe
Report Id: 37d63d3b-576b-4d9f-81b1-ea757344e448
Faulting package full name: 
Faulting package-relative application ID: 
```

According to glennsl at discord:
"from the logging it seems to crash here: https://github.com/revery-ui/revery/blob/ced2dde461c316178d4ce320b862a1c9d46a96a6/src/Core/Window.re#L413-L415"

I tried to compile simple SDL2 example and simple printf for GL_VERSION, GL_VENDOR and GL_SHADING_LANGUAGE_VERSION returns :
```OpenGL 1.1.0, Version Microsoft Corporation, GLSL: (null)```

**Comment by Samonitari** [comment]
Could it be that this is related to an empty window when starting oni on remote pc?
if I use rdp I get empty window, but not if I use Teamviewer?

**Comment by bryphe** [comment]
Hi @pa3k-lu - this should be fixed now with #2283 . We use ANGLE (Direct3D) instead of OpenGL, which tends to work better under virtualization. Let us know if you still are seeing issues launching!

**Comment by Samonitari** [comment]
This indeed solved the remote desktop issue as well for me!
